lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -41,11 +30,25 @@
 			<div class="richListItem__content">
 				<slot name="content" />
 			</div>
-			<div v-if="$slots.meta" class="richListItem__metaData -hideOnMobile">
-				<slot name="meta" />
-			</div>
-			<div v-if="$slots.trailing" class="richListItem__trailingSlot">
-				<slot name="trailing" />
+			<div class="richListItem__rightContainer">
+				<div v-if="$slots.meta" class="richListItem__metaData -hideOnMobile">
+					<slot name="meta" />
+				</div>
+				<div v-if="$slots.actions" class="richListItem__actionSlot">
+					<ds-divider
```

Comment:
Wouldn't it be simpler if both dividers were directly under `__rightContainer`? The gap would be `$space-xxs` between each of the children and only checkbox would have additional `$space-xxs` right padding

Reply:
The whole meeting was a discussion about the fact that it can't be like that, it has to be in a container, because there may be more dividers there.

Reply:
I still don't get it, apparently. But ok. Let it be